assets/js/googlesitekit/datastore/user/constants.js

```diff
@@ -71,3 +71,6 @@ export const keyMetricsGA4Widgets = [
 	KM_ANALYTICS_TOP_COUNTRIES,
 	KM_ANALYTICS_TOP_TRAFFIC_SOURCE,
 ];
+
+export const KM_CONNECT_GA4_CTA_WIDGET_DISMISSED_ITEM_KEY =
+	'key-metrics-connect-ga4-cta-widget';
```

Comment:
Let's also move this with GA4 constants.

Reply:
Moved, thank you!